sql/_27_banana_qa/issue_12366_DDL_Create_Table_If_Not_Exists/cases/03_if_not_exists_bug_from_web_001.sql

```diff
@@ -111,26 +111,26 @@ select * from t2 order by 1;
 DROP TABLE if exists t1,t2;
 CREATE TABLE t1 (id INT NOT NULL);
 INSERT INTO t1 VALUES (1),(1);
-CREATE TABLE IF NOT EXISTS t2 SELECT * FROM t1;
+CREATE TABLE IF NOT EXISTS t2; SELECT * FROM t1;
 
 DROP TABLE if exists t1,t2;
 
 DROP TABLE if exists t1,t2;
 CREATE TABLE t1 (id INT NOT NULL);
 INSERT INTO t1 VALUES (1),(1);
-CREATE TABLE IF NOT EXISTS t1 SELECT * FROM t1;
+CREATE TABLE IF NOT EXISTS t1; SELECT * FROM t1;
 
 DROP TABLE if exists t1,t2;
 
 DROP TABLE IF EXISTS ttt;
-CREATE TABLE IF NOT EXISTS ttt SELECT 0 AS n;
-CREATE TABLE IF NOT EXISTS ttt SELECT 0 AS n;
+CREATE TABLE IF NOT EXISTS ttt; SELECT 0 AS n;
+CREATE TABLE IF NOT EXISTS ttt; SELECT 0 AS n;
```

Comment:
CREATE TABLE IF NOT EXISTS ttt AS SELECT 0 AS n;
CREATE TABLE IF NOT EXISTS ttt AS SELECT 0 AS n;

Reply:
This review has been reflected.